apps/aecore/test/aec_chain_state_tests.erl

```diff
@@ -132,35 +139,35 @@ basic_access_test_block_chain() ->
     ?assertEqual(BH2, top_header(State2)),
 
     %% Check highest block
-    ?compareBlockResults({ok, B0}, {ok, top_block(State0)}),
-    ?compareBlockResults({ok, B1}, {ok, top_block(State1)}),
-    ?compareBlockResults({ok, B2}, {ok, top_block(State2)}),
+    ?compareBlockResults({ok, B0}, {ok, top_block(State0)}), %% TODO Check state trees too.
```

Comment:
Remove these TODOs as they were never part of this test. The macro removes the state anyway, and the root hash is already compared.